src/ufo.rs

```diff
@@ -194,6 +193,39 @@ impl Ufo {
                 .collect();
             let layers = layers?;
 
+            match meta.format_version {
+                FormatVersion::V1 | FormatVersion::V2 => {
+                    // Upconvert UFO v1 or v2 kerning if necessary. To upconvert, we need at least
+                    // a groups.plist file...
+                    if let Some(groups_to_upconvert) = &groups {
+                        // ... while a kerning.plist is optional. If there isn't one, pass an empty
+                        // map to the converter to keep the code simple.
+                        let kerning_to_upconvert;
```

Comment:
I would just write this as,

```rust
// maybe just 'let kerning = ..'?
let kerning_to_upconvert = kerning.unwrap_or_default();
```

(if default is unavailable you can do `kerning.unwrap_or_else(|| BTreeMap::new()`)